src/_nebari/upgrade.py

```diff
@@ -555,7 +555,7 @@ def _version_specific_upgrade(
         # which they can override if they are happy they understand the situation.
         config["prevent_deploy"] = True
 
-        # Nebari version 2023.9.1 upgrades JupyterHub to 3.1.  CDS Dashboards are only compatible with
+        # Nebari version 2023.10.1 upgrades JupyterHub to 3.1.  CDS Dashboards are only compatible with
         # JupyterHub versions 1.X and so will be removed during upgrade.
         rich.print("\n ⚠️  Deprecation Warning ⚠️")
         rich.print(
```

Comment:
Can we add another deprecation warning:

--> We will be removing and ending support for ClearML, Prefect and kbatch in the next release. The kbatch has been functionally replaced by Argo-Jupyter-Scheduler. We have seen little interest in ClearML and Prefect in recent years, and removing makes sense at this point. However if you wish to continue using them with Nebari we encourage you to [write your own Nebari extension](https://www.nebari.dev/docs/how-tos/nebari-extension-system#developing-an-extension).

Reply:
Yep - added